modules/core/include/opencv2/core.hpp

```diff
@@ -300,19 +300,19 @@ function does not copy src itself but simply constructs the border, for example:
 @endcode
 @note When the source image is a part (ROI) of a bigger image, the function will try to use the
 pixels outside of the ROI to form a border. To disable this feature and always do extrapolation, as
-if src was not a ROI, use borderType | BORDER_ISOLATED.
+if src was not a ROI, use borderType | cv::BORDER_ISOLATED.
```

Comment:
`cv::` for automatic links generation is widely used now ([example](https://github.com/opencv/opencv/blob/4ed40fd6946269ea36c58e5c76a277e7a871c269/modules/core/include/opencv2/core.hpp#L257))

BTW, `#BORDER_ISOLATED` should also create proper link, but IMHO "cv::" prefix looks more natural than "#".